vip-application-importer/src/main/java/fr/insalyon/creatis/vip/applicationimporter/client/view/applicationdisplay/VIPLayout.java

```diff
@@ -178,4 +190,7 @@ public void onSuccess(List<String> result) {
     public String getTag() {
         return tagsCbItem._getValue().toString();
     }
+    public String getExecutionType(){
+        return executionTypeItem._getValue().toString();
```

Comment:
Could you verify what happens if no value is selected in the combobox ?

Reply:
Up : could you verify this field is validated ?

Reply:
Nothing happens, it get stuck in the creating loading pop up.